RELEASE.md

```diff
@@ -8,6 +8,18 @@
 
 In the below example, we will pretend that we're upgrading from `0.11.0` to `0.12.0`. Please use the CORRECT release numbers as these are just example numbers.
 
+## Release timeline
+
+Usually we start a release process on either Monday or Tuesday and finish by Friday.
```

Comment:
There are some discrepancies, though. For now we have been smoke testing after github release for particular version was available. and last couple of releases were done usually starting Tue/Wednesday - that is just a time shift, no problem. But It would be much better as you are describing right now. although there is no code freeze system that would prevent merging changes during release and testing given bits, so the tested software differs from what is gonna be released.

Reply:
Maybe just a bit more text around "this is what a typical release week might look like"? We don't stick to a strict schedule, depends what timezone the releaser is in, what other issues come up, etc, might be more clear this isn't prescriptive.